contrib/cni-plugins/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine
+ARG TARGETARCH
+ARG CNI_PLUGINS_RELEASE
+
+LABEL source_repository=https://github.com/sapcc/kubernikus/tree/master/contrib/cni-plugins
+WORKDIR /cni-plugins
+RUN mkdir -p /cni-plugins && \
+    wget -O- https://github.com/containernetworking/plugins/releases/download/${CNI_PLUGINS_RELEASE}/cni-plugins-linux-${TARGETARCH}-${CNI_PLUGINS_RELEASE}.tgz | tar -xz
```

Comment:
@databus23 would it be ok to also add multus cni here?

Reply:
nvm, created a separate image for it https://github.com/sapcc/kubernikus/commit/b5827c368b796c181b41d738c4bdcef99529b656